non-critical-infra/hosts/hetzner-01/default.nix

```diff
@@ -0,0 +1,64 @@
+{ config, pkgs, lib, inputs, ... }:
+
+{
+  imports =
+    [
+      ./hardware.nix
+      inputs.srvos.nixosModules.server
+      inputs.srvos.nixosModules.hardware-hetzner-online-amd
+    ];
+
+
+  deployment = {
+    targetHost = "65.109.26.213";
+    targetUser = "root";
+  };
+
+  # Bootloader.
+  boot.loader.grub.enable = true;
+  boot.supportedFilesystems = [ "zfs" ];
+  boot.loader.grub.mirroredBoots = [
+    { path = "/boot-1"; devices = [ "/dev/disk/by-id/nvme-SAMSUNG_MZQL23T8HCLS-00A07_S64HNJ0T508051" ]; }
+    { path = "/boot-2"; devices = [ "/dev/disk/by-id/nvme-SAMSUNG_MZQL23T8HCLS-00A07_S64HNJ0T508053" ]; }
+  ];
+  boot.loader.grub.useOSProber = true;
+
+
+  networking.hostName = "hetzner-01"; # Define your hostname.
+  networking.hostId = "745b334a";
+
+  disko.devices = import ./disko.nix;
+
+  # Set your time zone.
+  time.timeZone = "Europe/Paris";
```

Comment:
Any reason for setting this to your local time and not to UTC? You could set TZ in your environment to `Europe/Paris` instead.

Reply:
I'd also set time for all machines to the same timezone. It makes later things like correlating logs easier.

Reply:
Yeah sorry that's indeed a miss on my side, I'll change that and redeploy.